File: src/main/java/io/endertech/util/helper/TextureHelper.java
package io.endertech.util.helper;

public class TextureHelper
{
    public static String metaToType(int meta)
    {
        if (meta == 0) return "Creative";
        else if (meta == 1) return "Redstone";
        else if (meta == 2) return "Resonant";

        return "Unknown";
    }
}

File: src/main/java/io/endertech/util/helper/TextureHelper.kt
package io.endertech.util.helper

object TextureHelper {
    fun metaToType(meta: Int): String {
        if (meta == 0)
            return "Creative"
        else if (meta == 1)
            return "Redstone"
        else if (meta == 2) return "Resonant"

        return "Unknown"
    }
}
